APIs_Drivers/UnbufferedSerial/main.cpp

```diff
@@ -0,0 +1,35 @@
+#include "mbed.h"
+
+
+// Create a DigitalOutput object to toggle an LED whenever data is received
+static DigitalOut led_serial_read(LED1);
```

Comment:
I would call the variable `led` as we do in other examples with the benefit that it is a shorter name.